src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsCC.jsx

```diff
@@ -78,7 +73,72 @@ export default function DetailsCC({
 }
 
 DetailsCC.propTypes = {
```

Comment:
What's the reason for explicitly define what an appointment looks like here? It seems like defining the appointment as 'required' should be enough since the shape is define in the appointments transformer.

Reply:
@vbahinwillit That's a good point. I was over thinking it. Reverting to previous.